### ISSUE TITLE
[iOS 13.X] Ionic app is not opening in iOS ^13

### ISSUE THREAD

**Issue by M2Costa** [ionic-team/ionic-v3#1062]
**I'm submitting a ...** 
[X ] bug report
[ ] feature request

**Current behavior:**
The app crashes wen oppened in a device that was upgrated to iOS ^13

**Expected behavior:**
The app was working on previous versions of the iOS but since the version 13 was launched it starts to crash in some devices. It was supposed to work as before the update.

**Steps to reproduce:**
Just open a ionic V3 app in a iOS device

**Comment by StefanRein** [comment]
Hey,

we tried iOS 13, 13.1 and 13.1.1. All of them work.

```
Ionic:

   Ionic CLI          : 5.2.8 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.2.2

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 8.0.0, browser 5.0.4, ios 4.5.5, osx 4.0.2, windows 6.0.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.1.3, cordova-plugin-ionic-webview 2.5.1, (and 14 other plugins)

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   Android SDK Tools : 26.1.1 (/Users/vetz/Library/Android/sdk)
   ios-deploy        : 2.0.0
   NodeJS            : v10.16.2 (/usr/local/bin/node)
   npm               : 6.10.0
   OS                : macOS Mojave
   Xcode             : Xcode 11.0 Build version 11A420a

```

Maybe some plugin is crashing.
Also maybe some Usage Description is missing, like NSCameraUsageDescription.

Check if your app works in the simulator.
If not open safari and attach the debugger and check if there are any error messages in safari or Xcode.

If not, it is maybe some UsageDescription. Then check the Xcode error messages when you deploy on your phone.

So in summary: It is working fine for iOS 13.x and it is an issue on your side.

Provide more information or a GitHub repository and I could look into it.

**Comment by visuddha** [comment]
> **I'm submitting a ...**
> [X ] bug report
> [ ] feature request
> 
> **Current behavior:**
> The app crashes wen oppened in a device that was upgrated to iOS ^13
> 
> **Expected behavior:**
> The app was working on previous versions of the iOS but since the version 13 was launched it starts to crash in some devices. It was supposed to work as before the update.
> 
> **Steps to reproduce:**
> Just open a ionic V3 app in a iOS device

Experiencing the same issue. Working in the simulator bit app crash on iOS 13.1.1 .

**Comment by bmeneghini** [comment]
We found the problem.

For some unknown reason, some plugins that we were using in our application requires Bluetooth permission in iOS 13.1.1, even if they don't use Bluetooth.

After requesting this, our application started to work again. I'll have to look at why iOS is now requiring this, but the problem is solved.

Thank you @StefanRein for the help.

**Comment by sacha67** [comment]
@bmeneghini Hello, I have exactly the same problem. 

Your solution works but my app does not require Bluetooth. 

Did you find who is requesting the Bluetooth permission ?

**Comment by bmeneghini** [comment]
Hey @sacha67 

I think it was some cordova-phonegap plugins, I can't remember exactly which ones.

**Comment by sacha67** [comment]
Thank you, I will try to find which one.
